src/AppBundle/Entity/SiteRepository.php

```diff
@@ -11,9 +12,18 @@ class SiteRepository extends EntityRepository
      * @param int $framework
      * @return array
      */
-    public function search( $name = null, $framework = null)
+    public function search($name = null, $framework = null)
```

Comment:
I am pretty use this function isn't used anywhere else, so you might as well delete it